style.css

```diff
@@ -0,0 +1,117 @@
+:root {
+    --black: #222426;
+    --gray-1: #525960;
+    --gray-2: #6a737c;
+    --gray-3: #9fa6ad;
+    --gray-4: #d7d9dc;
+    --gray-5: #e4e6e8;
+    --gray-6: #f1f2f3;
+    --white: #ffffff;
+    --white-gray: #f8f9f9;
+    --navy: #39739d;
+    --dark-blue: #0074cc;
+    --blue: #0a95ff;
+    --lightblue: #e1ecf4;
+    --primary-orange: #f48225;
+}
+
+/* nav bar up */
+
+body {
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    margin: 0;
+}
+
+.search-icon {
+    font-size: inherit;
+    position: absolute;
+    left: 1em;
+    bottom: 0.7em;
+    width: 1em;
+    fill: var(--gray-2);
+}
+
+form {
+    position: relative;
+}
+
+nav {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    background-color: var(--white-gray);
+    box-shadow: rgba(0, 0, 0, 0.2) 0 0 5px 0px;
+    border-top: var(--primary-orange) solid 3px;
+    font-size: 15px;
+    column-gap: 0.7em;
+
+}
+
+nav a {
+    text-decoration: none;
+    color: var(--gray-2);
+
+}
+
+nav p {
+    color: var(--gray-2);
+}
+
+
+form input {
+    font-size: inherit;
+    border-radius: 0.3em;
+    background-color: var(--white);
+    border: 1px solid var(--gray-3);
+    padding-left: 2em;
+    width: 800px;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5em;
+
+}
+
+
+
+.login-button {
+    border: var(--navy) solid 1px;
+    background-color: var(--lightblue);
+    padding: 0.3em 0.3em 0.3em 0.5em;
+    border-radius: 0.2em;
+    color: var(--navy);
+    margin-right: 0.3em;
+
+}
+
+.signup-button {
+    border: var(--blue) solid 1px;
+    background-color: var(--blue);
+    color: var(--white);
+    padding: 0.3em 0.3em 0.3em 0.5em;
+    border-radius: 0.2em;
```

Comment:
These are not good class names. Let's say you need to add a new button into the page, which looks like the `Sign Up` one, but it's text is `Contact Us`. Are you gonna put the `signup-button` class? It will be confusing right, having a contact button with the signup class.

That's why, classes shouldn't care about the content, but the styles. Meaning, we should not have a `login-button` and a `signup-button` classes. Instead we should have a `blue-button` and `light-blue-button`.

This way, we can reuse those classes, no matter the content.